refarch-eai/pom.xml

```diff
@@ -15,6 +15,7 @@
     <version>0.0.1-SNAPSHOT</version>
     <name>refarch_eai</name>
     <description>This is the default starting project for a EAI project based on the reference architecture of it@M</description>
+    <url>${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}</url>
```

Comment:
I just looked inside the generated jar inside the CI where still the env var placeholders are present, which for me means this doesn't work at all.

The original change came from https://github.com/it-at-m/refarch-templates/pull/423#discussion_r1818838688 `scm` attribute

Reply:
Is there a project which used the vars and made a release to maven central?

Reply:
I would fork the repo make a release with deployment and look if everything is working. then i would delete the pushed package in maven central.

unfortunately i don't have the permission to create a new repo in the github organisation it@m.

@darenegade  is reponsible for mavencentral and github

Reply:
@darenegade wie schaut es mit dem Testen aus?

Reply:
@hupling I'm not the only person who manages our Github orga: https://github.com/orgs/it-at-m/people?query=role%3Aowner

If you need something, please don't imply it, but write your request to me specifically or go the regular way described in our OSPO wiki. Thank you

Reply:
here you said https://git.muenchen.de/ccse/cicd/sps/-/issues/632#note_882369

I should ask you if I do something with maven central.


@simonhir we can't test it because you can not delete artifacts in maven central see here https://git.muenchen.de/ccse/ospo/-/issues/539#note_907673

Reply:
@hupling You cannot delete **published/released** packages. There are multiple stages in maven central. 
As long as you don't release/publish, you can test and remove it.

Reply:
@hupling and i checked that with https://central.sonatype.com/artifact/de.muenchen.oss.mobidam/mobidam-eai-commons-starter and as visible via the "Source Control" link the replacement doesn't work.

I will replace them here with static text and mention it as TODO in the getting started docs.